### PR TITLE
Removes rkt as an orchestrator in master dir

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Removes rkt as an orchestrator (as agreed with everyone so far, including PM: not supported)
- Moves info about how to run calico/node as a rkt container into asservice.md.
- Adds `no-canonical: true` tags to ease the future releasing process

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: _data/master/navbars/usage.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ toc:
     path: /usage/calicoctl/install-and-configuration
   - title: calico/ctl container
     path: /usage/calicoctl/container
-- title: Calico and Systemd
+- title: Running Calico as a system service
```
Comment (caseydavenport) [inline _data/master/navbars/usage.yml]: "Running Calico as a _systemd_ service" might be more accurate?

File: master/usage/configuration/as-service.md [hunk 2]
```diff
@@ -1,19 +1,23 @@
 ---
-title: Running Calico Node Container as a Service
+title: Running calico/node as a service
```
Comment (caseydavenport) [inline master/usage/configuration/as-service.md]: If we're going to change this, "as a service" should probably be "as a systemd service"

Comment (caseydavenport) [reply master/usage/configuration/as-service.md]: service is a highly overloaded term.

File: _data/master/navbars/usage.yml (hunk 1 above)
Comment (emanic) [reply _data/master/navbars/usage.yml]: I know what you mean, to me, it resonates more "systemd"...but in the text, it says you don't need to use systemd, you can use "init daemons such as upstart". I do not have a strong opinion on this but was just trying to stay in sync with the doc. To me, "systemd" resonates more but WDYT

File: master/usage/configuration/as-service.md (hunk 2 above)
Comment (emanic) [reply master/usage/configuration/as-service.md]: I know what you mean, if you think we're OK to go to "systemd service" (despite this bit about the init daemon), I am good with it also.

File: master/usage/configuration/as-service.md [hunk 3]
```diff
@@ -118,3 +124,39 @@ The script will also stop the calico-node container when the service is stopped.
 > Be sure to check this before starting the service.
 {: .alert .alert-info}
 
+
+## Running calico/node in a rkt container
+
+Each Calico-rkt enabled node requires the `calico/node` container to be running.
+
+The `calico/node` container can be run directly through rkt and needs to be run as
+as fly stage-1 container.
```
Comment (bcreane) [inline master/usage/configuration/as-service.md]: as `a` fly stage-1 container?

File: _data/master/navbars/usage.yml (hunk 1 above)
Comment (caseydavenport) [reply _data/master/navbars/usage.yml]: Maybe we just need to call it "Running Calico with an init system"

File: master/usage/configuration/as-service.md (hunk 3 above)
Comment (emanic) [reply master/usage/configuration/as-service.md]: Good point. Ack, will correct this.

**Comment by emanic** [comment]
@bcreane I have not been able to verify the rkt instructions. In talking with @caseydavenport , we've identified that as a non-goal for the alpha, but had planned a follow up story for an engineer, but we weren't sure when any engineers would have the time for it. The main goal here was not to lose the information that we had already....but I'm not sure we can take on any improvements to it, at least for now. Happy to discuss this further, as well.

File: _data/master/navbars/usage.yml (hunk 1 above)
Comment (emanic) [reply _data/master/navbars/usage.yml]: We _could_ do that, but I had started to lean heavily towards "systemd", I did a little Googling and I think that systemd has achieved prominence. Apparently, there were some wars on this, but that was back in _2014_.... https://www.tecmint.com/systemd-replaces-init-in-linux/ But I am flexible on this, let me know your thoughts.

**Comment by bcreane** [comment]
@emanic - have you created a jira issue to track `rkt` deployment validation? Otherwise, LGTM!

**Comment by emanic** [comment]
@bcreane Thanks! I will create this JIRA to make sure we don't lose track of this.

File: master/usage/configuration/as-service.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
 ---
-title: Running Calico Node Container as a Service
+title: Running calico/node as a systemd service
```
Comment (caseydavenport) [inline master/usage/configuration/as-service.md]: Why don't we make this "Running calico/node with an init system"?

**Comment by caseydavenport** [comment]
One comment on the title but then LGTM

File: master/usage/configuration/as-service.md (hunk 4 above)
Comment (emanic) [reply master/usage/configuration/as-service.md]: Works for me.

**Comment by emanic** [comment]
Thanks you guys!

**Pull request merged**